request_a_govuk_domain/request/init_guidance/registrant_permission.md

```diff
@@ -0,0 +1,7 @@
+How to check:
+
+* Check that the person giving the permission is identified by name and identifies themselves as a C-suite officer (CIO, CTO, CEO, CFO, CISO etc)
```

Comment:
Abbreviations are supported in python-markdown. See https://python-markdown.github.io/extensions/abbreviations/